lib/adyen/rest.rb

```diff
@@ -29,7 +29,8 @@ def self.client
       Adyen::REST::Client.new(
         Adyen.configuration.environment,
         Adyen.configuration.api_username,
-        Adyen.configuration.api_password
+        Adyen.configuration.api_password,
+        Adyen.configuration.default_api_params
```

Comment:
I didn't know it was left out intentionally. 

I like the principle, in our case the merchant account never change, so it feel more like something I'd set once and forget, instead of setting it explicitly with each request.